Phocalstream_Web/Controllers/AccountController.cs

```diff
@@ -314,12 +314,24 @@ public ActionResult DeleteUserCollections()
             return RedirectToAction("UserCollections", "Account");
         }
 
-        public ActionResult EditUserCollection(long collectionID)
+        public ActionResult EditUserCollection(long collectionID, int count = -1)
         {
             EditUserCollection model = new EditUserCollection();
             model.Collection = CollectionRepository.First(c => c.ID == collectionID, c => c.Photos);
             model.CoverPhotoId = model.Collection.CoverPhoto != null ? model.Collection.CoverPhoto.ID : 0;
 
+            if (count != -1)

```

Comment:
It isn't a big deal but this conditional feels clunky. Why not

if ( count == 1) {
 ... 
} else if ( count > 0) {
 ...
}